en_US.json

```diff
@@ -1064,17 +1071,26 @@
 	"hint.directive.proxy_pass_request_headers": "pass request headers from client to upstream",
 	"hint.directive.proxy_redirect": "replace the 30x Location header",
 	"hint.directive.proxy_request_body_in_cache_key": "add req body in cacke key",
+	"hint.directive.proxy_request_buffering": "enable or disable request buffering",
 	"hint.directive.proxy_set": "assign value to a variable based on response",
 	"hint.directive.proxy_ssl_protocols": "TLS protocols to origin",
+	"hint.directive.range_reorder": "reorder and coalesce multiple ranges",
 	"hint.directive.realtime_log_downsample": "set the downsample factor",
 	"hint.directive.sanitize_accept_encoding": "preprocess Accept-Encoding header",
 	"hint.directive.satisfy": "how to combine multiple auth results",
 	"hint.directive.secure_link": "set up a built-in access control method",
 	"hint.directive.secure_link_md5": "set up a built-in access control method",
 	"hint.directive.secure_link_secret": "set up a built-in access control method",
+	"hint.directive.set_attack_state_cpu": "sets threshold for specified $server_name and random number between mintime and maxtime as expiration for resetting of the variable attack_state_cpu state",
+	"hint.directive.set_attack_state_req": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_req state",
+	"hint.directive.set_attack_state_rspbytes": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable",
+	"hint.directive.set_attack_state_tlshs": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_tlshs state",
 	"hint.directive.slice": "request big object in small pieces from origin",
 	"hint.directive.slice_ignore_etag": "allow slices to have different ETag values",
+	"hint.directive.slice_verify_header": "specify extra header to verify slice consistency in additional to etag",
```

Comment:
Should be "in addition to etag" and not "in additional to etag"

Reply:
I modified it. Thanks.